app/assets/javascript/components/searchable_collection_component/searchable_collection.js

```diff
@@ -0,0 +1,69 @@
+import { Controller } from '@hotwired/stimulus';
+
+const SearchableCollectionComponent = class extends Controller {
```

Comment:
This should probably be called `Controller` not `Component` for consistency?

Reply:
gonna do this all in another PR, there is more inconsistency than just this